feat: Allow skipping Wave server TLS verification #1547 (#1556)
h2oai/wave: website/docs/configuration.md

H2O_WAVE_TLS_CERT_FILE
H2O_WAVE_TLS_KEY_FILE
H2O_WAVE_WEB_DIR
H2O_WAVE_VERIFY_TLS
```

Notes:

- [1] `1`, `t`, `true` to enable; `0`, `f`, `false` to disable (case insensitive).
- [2] Use OS-specific path list separator to specify multiple arguments - `:` for Linux/OSX and `;` for Windows. For example, `H2O_WAVE_PUBLIC_DIR=/images/@./files/images:/downloads/@./files/downloads`.

### TLS verification

During development, you might want to test out TLS encryption, e.g. communication between Wave server and Keycloak. The easiest thing to do is to generate a self-signed certificate. However, Wave server verifies certificates for all communication by default, thus would throw an error for a self-signed one. ***FOR DEVELOPMENT PURPOSES ONLY***, it's possible to turn off the check using either `H2O_WAVE_VERIFY_TLS` environment variable or `verify-tls` parameter.

:::warning

> Make sure the TLS verification is not turned off in production environments as it's a huge security hole.

Reviewer: ```suggestion
**Disabling TLS verification is a security risk.** Make sure TLS is not disabled in production environments.
```
